3rd_party_slots/rust_metta_bus_client/README.md

```diff
@@ -30,17 +30,17 @@ If you need to setup a local DAS stack, follow #[] before
 
 After building you can run it by:
 ```sh
-./target/release/metta-bus-client localhost:8080 localhost:35700 0 0 'LINK_TEMPLATE Expression 3 NODE Symbol Similarity NODE Symbol "human" VARIABLE S'
+./target/release/metta-bus-client localhost:8080-8090 localhost:40002 0 0 'LINK_TEMPLATE Expression 3 NODE Symbol Similarity NODE Symbol "human" VARIABLE S'
```

Comment:
I understand this is just a README but you are advising the user to use 8080 which is a "reserved" port and may lead to unexpected errors.